### PR TITLE
added product serializer

### PR DESCRIPTION
**Pull request by danirenae**
Added a product serializer for review

## Description
The product serializer is based on the product model and should serialize the product model.

## Related Issue
This is related to Issue #32 

## Motivation and Context
This creates a serializer for the Product model to easily serialize the data.

## How Has This Been Tested?
This was tested in the terminal.

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help! 
- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: bangazon/bangazon_api/serializers.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from bangazon_api.models import Customer, ProductType, PaymentType, Order, Product
+from rest_framework import serializers
+
```
Comment (mccordgh) [inline bangazon/bangazon_api/serializers.py]: 💩

File: bangazon/bangazon_api/serializers.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from bangazon_api.models import Customer, ProductType, PaymentType, Order, Product
```
Comment (mccordgh) [inline bangazon/bangazon_api/serializers.py:1]: 👀

File: bangazon/bangazon_api/serializers.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+from rest_framework import serializers
+
+
+class ProductSerializer(serializers.HyperlinkedModelSerializer):
```
Comment (rtwhitfield84) [inline bangazon/bangazon_api/serializers.py:5]: add doc strings

Comment (mccordgh) [inline bangazon/bangazon_api/serializers.py:5]: Looking for docstream 👁

**Review by mccordgh — CHANGES_REQUESTED** [review]
Here is my review. Thanks. Testing 1 2 3

File: bangazon/bangazon_api/serializers.py [hunk 4]
```diff
@@ -3,6 +3,10 @@
 
 
 class ProductSerializer(serializers.HyperlinkedModelSerializer):
+  """
+    Hyperlinked Serializer for the Product Model.
```
Comment (mccordgh) [inline bangazon/bangazon_api/serializers.py:7]: looks good 👍

**Review by mccordgh — APPROVED** [review]

**Review by rtwhitfield84 — APPROVED** [review]
👍

**Pull request merged**